san/sanbase_graphql.py

```diff
@@ -287,8 +287,15 @@ def _format_return_fields(return_fields):
     return ",\n".join(return_fields)
 
 
-def _format_date(datetime_obj_or_str):
+def _format_from_date(datetime_obj_or_str):
     if isinstance(datetime_obj_or_str, datetime.datetime):
         datetime_obj_or_str = datetime_obj_or_str.isoformat()
 
     return iso8601.parse_date(datetime_obj_or_str).isoformat()
+
+def _format_to_date(datetime_obj_or_str):
+    if isinstance(datetime_obj_or_str, datetime.datetime):
+        datetime_obj_or_str = datetime_obj_or_str.isoformat()
+
+    dt = iso8601.parse_date(datetime_obj_or_str) + datetime.timedelta(hours=23, minutes=59, seconds=59)
```

Comment:
Why is the end of the day taken here?

Reply:
Seems to me that the end of the day is more appropriate here otherwise it includes this day but without the full data for the day. You can check the examples here: https://github.com/santiment/sanpy/issues/39